.github/workflows/earthly-build.yaml

```
@@ -51,4 +56,4 @@ jobs:
         run: |
           echo $DOCKER_HUB_TOKEN | docker login --username osrfbot --password-stdin
           cd spaceros
-          earthly --ci --push +image
+          earthly --ci --push +image --tag=$INPUT_IMAGE-TAG
```

I think we want to [use Actions's interpolation](https://docs.github.com/en/actions/learn-github-actions/contexts#example-usage-of-the-inputs-context-in-a-manually-triggered-workflow) here instead of environment variables:
```suggestion
          earthly --ci --push +image --tag=${{ inputs.image-tag }}
```

---

I guess I shouldn't be surprised that the Actions docs are self-conflicting!